src/main/java/Exercise8.java

```diff
@@ -0,0 +1,13 @@
+public class Exercise8 {
+    public int factorial(int n){
+        int p = 1;
+        if(n==0||n==1)
```

Comment:
Câu lệnh if else này có thể bỏ đi. cứ đưa đại vòng for vào thì vẫn không thay đổi kết quả

Reply:
Done!